Gemfile

```diff
@@ -3,7 +3,6 @@ source 'https://rubygems.org'
 ruby File.read(".ruby-version").strip
 
 gem 'gds-api-adapters', '~> 47.9'
-gem 'govuk_ab_testing', '~> 2.4'
```

Comment:
You need to commit the updated Gemfile.lock file as well. This is why the Heroku deployment is failing.